BoxInBox/test/levels1.par

```diff
@@ -3,6 +3,7 @@ ActiveThorns = "
     CarpetX
     CoordinatesX
     IOUtil
+    CarpetXRegrid
```

Comment:
Can you sort thorn names alphabetically?

Reply:
yes